<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="On_LoadFactor_16">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>Load Factor 16</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_LoadFactor_64">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>Load Factor 64</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_LoadFactor_128">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>Load Factor 128</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_LoadFactor_2048">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 147, 0);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>Load Factor 2048</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar_2">
          <property name="styleSheet">
           <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    background-color: #0000FF;
}

QProgressBar::chunk {
    background-color: #05B8CC;
    width: 20px;
}
</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="styleSheet">
           <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    background-color: #FF0000;
}

QProgressBar::chunk {
    background-color: #05B8CC;
    width: 20px;
}
</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>543</width>
          <height>511</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
